lib/calculator.rb

```diff
@@ -0,0 +1,15 @@
+module Calculator
+
+  def subtract(a, b)
+    a - b
+  end
+
+  def divide(a, b)
+    a / b
+  end
+
+  def add(a, b)
+    a + b
+  end
+
+end
```

Comment:
This module saves us a lot of space especially as it is used throughout both classes.